solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,211 @@
+const {contract, web3, accounts} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {time} = require("@openzeppelin/test-helpers")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const WrappedToken = contract.fromArtifact("TestToken")
+
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const expect = chai.expect
+
+describe("LPRewards", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+  const allocationForDistribution = web3.utils.toWei("10000000")
```

Comment:
Nitpick: I'd rather express it as `web3.utils.toBN("10000000").mul(tokenDecimalMultiplier)` given that we are not transforming ether to wei but KEEP into 1e18 format. Same in all other places in this file where we use `toWei` but we are dealing with KEEP.

Reply:
okay, reverted to the original version